pkgs/desktops/gnome-3/misc/gnome-flashback-session/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, gnome3, bash, glib, wrapGAppsHook }: { wmName, wmLabel, wmCommand }:
+stdenv.mkDerivation {
+  name = "gnome-flashback-${wmName}";
+
+  buildInputs = [ gnome3.gnome-flashback gnome3.gnome-panel bash glib ];
+  nativeBuildInputs = [ wrapGAppsHook ];
+
+  unpackPhase = "true";
+
+  installPhase = ''
+    mkdir -p $out/libexec
+    cat << 'EOF' > $out/libexec/gnome-flashback-${wmName}
+    #!${bash}/bin/sh
+
+    if [ -z $XDG_CURRENT_DESKTOP ]; then
+      export XDG_CURRENT_DESKTOP="GNOME-Flashback:GNOME"
+    fi
+
+    exec ${gnome3.gnome-session}/bin/gnome-session --session=gnome-flashback-${wmName} "$@"
```

Comment:
Maybe add an option for extra arguments like the acceleration disablement.

Reply:
Here is a (commit adding this flag](https://gitlab.gnome.org/GNOME/gnome-flashback/commit/39dbfd09d4a7e16d5ca41319a62f173b8c79ca79) and a [relevant bug](https://bugzilla.gnome.org/show_bug.cgi?id=731173).